src/services/SettingsAPI.ts

```diff
@@ -22,24 +23,25 @@ type TalkSettingsSection = {
 	element: string
 }
 
-const customSettingsSections: TalkSettingsSection[] = shallowReactive([])
+// TODO: use shallowReactive instead of ref + markRaw in Vue 3 (see file commit history)
+const customSettingsSections: Ref<TalkSettingsSection[]> = ref([])
```

Comment:
`shallowReactive` has no real advantages except that it allows to get rid of `markRaw` in just one function.